pages/index.js

```diff
@@ -61,15 +61,18 @@ export default class Index extends React.Component {
       OS = 'linux'
     }
 
-    const releases = await cachedFetch(
-      `https://api.github.com/repos/zeit/hyper/releases`,
-      {
-        headers: {
-          Authorization: `token ${process.env.GITHUB_TOKEN}`
-        }
-      },
-      'json'
-    )
+    // const releases = await cachedFetch(
+    //   `https://api.github.com/repos/zeit/hyper/releases`,
+    //   {
+    //     headers: {
+    //       Authorization: `token ${process.env.GITHUB_TOKEN}`
+    //     }
+    //   },
+    //   'json'
+    // )
+    if (res) {
+      res.setHeader('Cache-Control', 'Cache-Control: s-maxage=7200')
```

Comment:
Add `, stale-while-revalidate` so the blocking request is always async

Reply:
Done 👍